src/core/public/chrome/doc_title/doc_title_service.ts

```diff
@@ -43,113 +41,56 @@ interface StartDeps {
  * @public
  * */
 export interface ChromeDocTitle {
-  /**
-   * Gets an observable of the current document title.
-   */
-  get$: () => Observable<string>;
   /**
    * Changes the current document title.
    *
-   * @remarks
-   * The apply option is mostly here for legacy compatibility reasons and it's default value
-   * should probably always be used.
-   *
    * @example
    * How to change the title of the document
    * ```ts
    * chrome.docTitle.change('My application title')
    * chrome.docTitle.change(['My application', 'My section'])
-   * chrome.docTitle.change({ parts: ['My application'], excludeBase: true })
    * ```
    *
-   * @param newTitle The new title to set, either a string, string array or {@link ChromeDocTitleEntry}
-   * @param [apply=true] If false, will not actually apply the new title until #apply() is called.
+   * @param newTitle The new title to set, either a string or string array
    */
-  change: (newTitle: ChromeDocTitleChange, apply?: boolean) => void;
+  change(newTitle: ChromeDocTitleChange): void;
   /**
    * Resets the document title to it's initial value.
    * (meaning the one present in the title meta at application load.)
-   *
-   * @remarks
-   * The apply option is mostly here for legacy compatibility reasons and it's default value
-   * should probably always be used.
-   *
-   * @param [apply=true] If false, will not actually apply the new title until #apply() is called.
    */
-  reset: (apply?: boolean) => void;
-  /**
-   * Apply a title change or reset that was not applied yet.
-   *
-   * @see {DocTitle#change}
-   * @see {DocTitle#reset}
-   */
-  apply: () => void;
+  reset(): void;
+
   /** @internal */
   __legacy: {
-    setBaseTitle: (baseTitle: string) => void;
+    setBaseTitle(baseTitle: string): void;
   };
 }
 
-/** @public */
-export interface ChromeDocTitleEntry {
-  parts: string[];
-  excludeBase?: boolean;
-}
-
 /**
  * Composed type for the {@link ChromeDocTitle#change} possible inputs.
  *
  * @public
  */
-export type ChromeDocTitleChange = string | string[] | ChromeDocTitleEntry;
+export type ChromeDocTitleChange = string | string[];
```

Comment:
optional: Is there any advantage to having a named type? It feels like it might be simpler if to inline if we're only using primitive types, but happy for you to merge this either way.

Reply:
I had the same question. I'm quite used to always use named types from were I worked before, but for that kind of scenario, I cant really think of any real advantage (expect maybe refactoring if one day we modify/extend the type), and I agree that I doesn't really makes documentation more clear. If we think this is better I definitely can fall back to inline types